### DISCUSSION TITLE
Not able to deploy app to iOS emulator using the remote Mac connection from Windows

### DISCUSSION THREAD

**Discussion by pabloquesadaglobant** [dotnet/maui#17660] [General]
I have an app for Android and iOS using .net Maui (.Net7, VS2022). When deploying the app to Android, it succeeded. But when trying to deploy to iOS it failed every single time. I'm using the next procedure:
1. I pair my Visual Studio (on Windows) to the network accesible Mac laptop.
2. I select the desired emulator in the "Run" button to use "iPhone 14 iOS 16.4".
3. I click on the "Run" button and the process started.

The build process didn't complete (and of course the deploy won't work) because the error that I show below. Something very important: we are pretty sure the code is fine and the Mac works to pair/deploy cause another work mate is doing the process successfully with exactly the same repo.

>clang: error: linker command failed with exit code 1 (use -v to see invocation)
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.7098\targets\Xamarin.Shared.Sdk.targets(1274,3): error : clang++ exited with code 1:
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.7098\targets\Xamarin.Shared.Sdk.targets(1274,3): error : ld: in /Users/build/Library/Caches/Xamarin/mtbs/builds/[...]), building for iOS Simulator, but linking in object file built for iOS, file '/Users/build/Library/Caches/Xamarin/mtbs/builds/[...]'
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.7098\targets\Xamarin.Shared.Sdk.targets(1274,3): error : clang: error: linker command failed with exit code 1 (use -v to see invocation)
2>Done building project "[...].csproj" -- FAILED.
========== Build: 1 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Build started at 11:03 AM and took 03:18.438 minutes ==========
========== Deploy: 0 succeeded, 0 failed, 0 skipped ==========

**Comment by maexsp** [comment]
Hi. 
Probably a VS2022 tooling issue and AppCenter issue. See https://github.com/dotnet/maui/issues/17236 or use a physical iPhone. 
BR
